apps/datahub/src/app/record/header-badge-button/header-badge-button.component.html

```diff
@@ -1,15 +1,5 @@
 <button
-  class="
-    items-center
-    justify-center
-    px-2
-    py-1
-    bg-primary
-    text-white
-    rounded
-    opacity-25
-    capitalize
-  "
+  class="items-center justify-center px-2 py-1 bg-primary text-white rounded opacity-25 capitalize"
```

Comment:
I don't understand why we have such changes...
Prettier or `npm run format` should guarantee to always have the correct indentation.

Reply:
Prettier has changed its strategy for html formatting with version 2.5 (https://prettier.io/blog/2021/11/25/2.5.0.html#collapse-html-class-attributes-onto-one-line-11827httpsgithubcomprettierprettierpull11827-by-jlongsterhttpsgithubcomjlongster), and does not wrap lines > `printWidth` anymore. I didn't see another param to conserve the old behavior, which might not make sense anyway, since this evolution.

Reply:
Yes but i wonder why all those changes were not automatic and mandatory for the CI when angular and prettier were updated.

Reply:
That's because prettier is only run on `affected` libraries and apps, so the new formatting strategy has not been applied everywhere but will be gradually. I guess it would be preferable to have all HTML reformatted correctly to avoid confusing diffs in the future.

Reply:
yes, at some point we should run test, lint and format on all sources to avoid having broken stuff while didn't change anything about them.
We should add npm scripts for that IMO.